### PR TITLE
ODIM-3389 Unit test cases for delete volume

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Review by bharath-b-hpe — APPROVED** [review]

**Review by shivacharanms — CHANGES_REQUESTED** [review]
Remove references of volume collection as this is not of the story assigned

File: svc-api/main.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func main() {
 		var basicAuthToken string
 		if basicAuth != "" {
 			var urlNoBasicAuth = []string{"/redfish/v1", "/redfish/v1/SessionService"}
-			var authRequired bool = true
+			authRequired := true
```
Comment (shivacharanms) [inline svc-api/main.go]: declare and define in separate statements

Comment (irappa-pattar) [reply svc-api/main.go]: Changed

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**